### PR TITLE
[WIP][DRAFT] SSL Certs

### PR DESCRIPTION
**Pull request by rahulkp220**
Vercel https://vercel.com/docs/projects/domains/working-with-ssl
Netlify https://docs.netlify.com/domains-https/https-ssl/#netlify-managed-certificates

### CODE REVIEW

File: rust-toolchain.toml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.74.1"
+channel = "stable"
```
Comment (rahulkp220) [inline rust-toolchain.toml:2]: @Alw3ys although an unrelated change, believe this could be helpful to always stay on the latest stable release.

Comment (Alw3ys) [reply rust-toolchain.toml:2]: I strongly believe is best to pin versions everywhere and setup a task to do it once a while. otherwise people will have different versions

**Comment by rahulkp220** [comment]
Superseded by https://github.com/doseiai/engine/pull/64
